loadflow/loadflow-api/src/main/java/com/powsybl/loadflow/LoadFlowParameters.java

```diff
@@ -85,7 +99,8 @@ public enum ConnectedComponentMode {
     // VERSION = 1.7 hvdcAcEmulation
     // VERSION = 1.8 noGeneratorReactiveLimits -> useReactiveLimits
     // VERSION = 1.9 dcPowerFactor
-    public static final String VERSION = "1.9";
+    // VERSION = 2.0 componentMode instead of connectedComponentMode
```

Comment:
Why do you change the version to 2.0? I think 1.10 would be better for this change.